iOS_SDK/OneSignalSDK/Source/OneSignal.h

```diff
@@ -469,7 +469,7 @@ extern NSString* const ONESIGNAL_VERSION;
 
 #pragma mark Initialization
 + (void)setAppId:(NSString* _Nonnull)newAppId;
-+ (void)setLaunchOptions:(NSDictionary* _Nullable)launchOptions;
++ (void)init:(NSDictionary* _Nullable)launchOptions;
```

Comment:
Should this be `initWithLaunchOptions:` to be more consistent with Objective-C naming? But we should also consider what it will look like for Swift too. If we go with `initWithLaunchOptions:` would it result in `OneSignal.init(launchOptions)` in Swift?